.github/workflows/lint-collection.yml

```diff
@@ -5,5 +5,3 @@ name: Ansible collection linters
 jobs:
   lint:
     uses: stackhpc/.github/.github/workflows/lint-collection.yml@main
-    with:
-      lint_pip_dependencies: "git+https://github.com/stackhpc/ansible-modules-hashivault@stackhpc"
```

Comment:
Getting this into the template: https://github.com/stackhpc/stackhpc-release-train/pull/230